driver.go

```diff
@@ -191,6 +234,31 @@ func (sumoDriver *sumoDriver) StopLogging(file string) error {
   return nil
 }
 
+func interpretAll(re *regexp.Regexp, input string, dictionary map[string]string) string {
+  result := ""
+  lastIndex := 0
+
+  for _, v := range re.FindAllSubmatchIndex([]byte(input), -1) {
+      groups := []string{}
+      for i := 0; i < len(v); i += 2 {
```

Comment:
Why is it incrementing by 2?

Reply:
it's because the return of the FindAllSubmatchIndex() actually return a pattern as begin/end index of the matching sections